mobile/analytics/partners.js

```diff
@@ -2,16 +2,17 @@
   'use strict'
 
   $('.partner-profile-contact-email a').click(function (e) {
-    analytics.track('Clicked Contact Gallery Via Email', {
+    analytics.track('Click',{
       partner_id: $(e.currentTarget).data('partner-id'),
-      partner_slug: $(e.currentTarget).data('partner-slug')
-    })
+      label: "Contact gallery by email"
+    });
```

Comment:
It seems like analytics.js send user agent by default and there's no need to send extra properties.